assets/global.js

```diff
@@ -1154,9 +1154,11 @@ class VariantSelects extends HTMLElement {
       });
     }
 
-    document
-      .querySelector(`[id^="MediaGallery-${this.dataset.section}"]`)
-      ?.setActiveMedia?.(`${this.dataset.section}-${this.currentVariant.featured_media?.id}`);
+    if (this.currentVariant.featured_media) {
+      document
+        .querySelector(`[id^="MediaGallery-${this.dataset.section}"]`)
+        ?.setActiveMedia?.(`${this.dataset.section}-${this.currentVariant.featured_media?.id}`);
```

Comment:
nit -- we should be able to remove the optional chain here `this.currentVariant.featured_media?.id` since we already check that featured_media is preesent